developer/src/kmc-kmn/test/fixtures/kmw/test_index_12980.kmn

```diff
@@ -0,0 +1,25 @@
+﻿c test_index_12980 generated from template at 2025-01-22 13:25:35
+c Validates index() offsets when used with if() - see #12980
+
+store(&VERSION) '14.0'
+store(&NAME) 'test_index_12980'
+store(&TARGETS) 'web'
+
+
+begin Unicode > use(main)
+
+group(main) using keys
+
+store(a) '1'
+store(b) '2'
+store(c) '3'
+store(d) '4'
+store(e) '5'
+store(f) '6'
+
+if(&platform = 'web') any(a) + 'a' > context 'A'
+if(&platform = 'web') any(b) + 'b' > context(2) 'B'
+if(&platform = 'web') notany(c) + 'c' > context 'C'
+if(&platform = 'web') notany(d) + 'd' > context(2) 'D'
+nul any(e) + 'e' > context 'E'
+nul any(f) + 'f' > context(2) 'F'
```

Comment:
Any value in also testing the combo
```
+ nul notany(g) + 'g' >  ...?
```

Reply:
Fun detail:  `notany(g)` can match `nul`.  I'm not 100% on if that holds true here, with `nul notany(g)`, but the Web-side implementation will happily match such a rule in that manner.

Reply:
> Any value in also testing the combo
> 
> ```
> + nul notany(g) + 'g' >  ...?
> ```

I don't think it adds any additional coverage but am happy to add it.